doc/proto.md

```diff
@@ -116,6 +117,25 @@ GenesisState - genesis state of x/wasm
 | codes | [Code](#cosmwasm.wasm.v1beta1.Code) | repeated |  |
 | contracts | [Contract](#cosmwasm.wasm.v1beta1.Contract) | repeated |  |
 | sequences | [Sequence](#cosmwasm.wasm.v1beta1.Sequence) | repeated |  |
+| gen_msgs | [GenesisState.GenMsgs](#cosmwasm.wasm.v1beta1.GenesisState.GenMsgs) | repeated |  |
+
+
+
+
+
+
+<a name="cosmwasm.wasm.v1beta1.GenesisState.GenMsgs"></a>
+
+### GenesisState.GenMsgs
+GenMsgs define the messages that can be executed during genesis phase.
```

Comment:
A mention they are executed in order, and how to reference contract addresses? Or I guess that goes in `x/wasm`

Reply:
I have added `list-contracts` and `list-code` commands that also display the queued message data. So it should be easy to find the contract address for any instantiation.

Reply:
That is very cool. So I write an init one, and then I can list them to see what the address was and use that result for future tx (or bank accounts).

Nice tooling